### ISSUE TITLE
Create service form fieldset has no accessible name

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#370]
>
This fieldset element does not have a name available to an accessibility API. Valid names are: legend element.

**Comment by msecret** [comment]
Closing to one issue.
